src/components/ArticleCard.jsx

```diff
@@ -1,24 +1,61 @@
-import { Card, Col } from "react-bootstrap";
+import { Card, Col, Button } from "react-bootstrap";
```

Comment:
Always love seeing the use of bootstrap, it's a good thing to get familiar with 👍